src/libm-tester/tester.c

```diff
@@ -29,7 +29,7 @@
 #include "testerutil.h"
 
 void stop(char *mes) {
-  fprintf(stderr, "%s\n", mes);
+  printf("%s\n", mes);
```

Comment:
No, no problem with that. But it is a change that should have been introduced via a different pull request.